modules/ibi-create-config-iso.adoc

```diff
@@ -0,0 +1,217 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/ibi-edge-image-based-install.adoc 
+
+:_content-type: PROCEDURE
+[id="create-config-iso_{context}"]
+= Generating a configuration ISO for {sno} clusters
+
+Create the following site-specific configuration resources in the hub cluster to generate a configuration ISO:
+
+* `BareMetalHost`
+* `ImageClusterInstall`
+* `ClusterDeployment`
+* `ClusterImageSet`
+* `ManagedCluster`
+
+The IBI Operator uses the configuration ISO to configure and deploy a host that you preinstalled with {sno}. 
+
+.Prerequisites
+
+* You preinstalled a host with {sno} using an image-based installation.
+* You logged in as a user with `cluster-admin` privileges.
+* You installed the IBI Operator on the hub cluster running {rh-rhacm-first}.
+
+.Procedure
+. Create the `BareMetalHost` and `Secret` resources:
+
+.. Create a YAML file that defines the `BareMetalHost` and `Secret` resources:
++
+.Example `ibi-bmh.yaml` file
+[source,yaml]
+----
+apiVersion: metal3.io/v1alpha1
+kind: BareMetalHost
+metadata:
+  name: ibi-bmh <1>
+  namespace: openshift-machine-api
+spec:
+  online: true <2>
+  bootMACAddress: 00:a5:12:55:62:64 <3>
+  bmc:
+    address: redfish-virtualmedia+http://192.168.111.1:8000/redfish/v1/Systems/8a5babac-94d0-4c20-b282-50dc3a0a32b5 <4>
+    credentialsName: ibi-bmh-bmc-secret <5>
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: ibi-bmh-secret <6>
+  namespace: openshift-machine-api
+type: Opaque
+data:
+  username: <user_name> <7>
+  password: <password> <8>
+----
+<1> Specify the name for the `BareMetalHost` resource.
+<2> Specify if the host should be online. 
+<3> Specify the host boot MAC address.
+<4> Specify the BMC address. You can only use bare-metal host drivers that support virtual media networking booting, for example redfish-virtualmedia and idrac-virtualmedia.
+<5> Specify the name of the bare-metal host `Secret` resource.
+<6> Specify the name for the `Secret` resource.
+<7> Specify the username.
+<8> Specify the password.
+
+.. Create the `BareMetalHost` and `Secret` resources by running the following command:
++
+[source,terminal]
+----
+$ oc create -f ibi-bmh.yaml
+----
+
+. Create the `ClusterImageSet` resource:
+
+.. Create a YAML file that defines the `ClusterImageSet` resource:
++
+.Example `ibi-cluster-image-set.yaml` file
+[source,yaml]
+----
+apiVersion: hive.openshift.io/v1
+kind: ClusterImageSet
+metadata:
+  name: ibi-img-version-arch <1>
+spec:
+  releaseImage: ibi.example.com:path/to/release/images:version-arch <2>
+----
+<1> Specify the name for the `ClusterImageSet` resource.
+<2> Specify the address for the release image to use for the deployment. 
+// Does this need to sync with the version used in the install or any requirements here?
```

Comment:
The only thing we use this for is `ReleaseRegistry` in the SeedReconfiguration.
@eranco74 how should we describe this? What is `ReleaseRegistry` used for?

Reply:
ReleaseRegistry is used [here](https://github.com/openshift-kni/lifecycle-agent/blob/8662ded562254bfc449fd109198489651ffa4835/lca-cli/postpivot/postpivot.go#L507), it updates the cluster-version-operator deployment to use the release image from the given registry in case it's different then the original registry the seed was created with
https://github.com/openshift-kni/lifecycle-agent/blob/8662ded562254bfc449fd109198489651ffa4835/lca-cli/postpivot/postpivot.go#L507

 ClusterImageSet defines an openshift release image that is available on the hub cluster and matches the OCP version in the seed image.

Reply:
Does this take a CIDR address? 

Just looking at the field as it appears in the install config:

````
networking:
  machineNetwork:
  - cidr: <public_cidr>
````